samples/sensor/hx711/src/main.c

```diff
@@ -0,0 +1,114 @@
+/*
```

Comment:
Shouldn't this file be part of example commit ? https://github.com/zephyrproject-rtos/zephyr/pull/24295/commits/4eb64a10e35e7f14fa9b835d9b22fb4d46ede369

Reply:
Right.